docs/source/examples.rst

```diff
@@ -53,16 +53,22 @@ can be used, called ``MoabCluster``:
 SGE Deployments
 ---------------
 
-On systems which use SGE as the scheduler, ``SGECluster`` can be used:
+On systems which use SGE as the scheduler, ``SGECluster`` can be used.
+
+SGE systems have a lot of flexibility in how they are configured, hence it is not possible to use the ``memory`` keyword argument to automatically specify the amount of RAM requested. Instead, you specify the resources desired according to how your system is configured, using the ``resource_spec`` keyword argument, in addition to the ``memory`` keyword argument (which is used by Dask internally for memory management).
+
+In the example below, our system administrator has used the ``m_mem_free`` keyword argument to let us request for RAM. At the same time, we must also correctly specify the ``memory`` keyword argument, to enable Dask's memory management to do its work correctly.
 
 .. code-block:: python
 
     from dask_jobqueue import SGECluster
 
     cluster = SGECluster(queue='default.q',
                          walltime="1500000",
-                         processes=10,
-                         memory='20GB')
+                         processes=10,   # we request 10 processes per worker
+                         memory='20GB',  # for memory requests, this must be specified
+                         resource_spec='m_mem_free=20G',  # for memory requests, this also needs to be specified
```

Comment:
maybe here or in the text above specify that `m_mem_free` is scheduler specific and that you should check your cluster doc or ask your sys admin if you are unsure. Also, if that makes sense, adding the most commonly used memory names would be a good idea (from a quick googling) could be a good idea.